check-packages.bash

```diff
@@ -8,8 +8,10 @@ cleanup() {
 }
 trap cleanup EXIT
 working_dir="$(mktemp --directory)"
+readonly working_dir
```

Comment:
Is it good form to combine assigment and setting readonly in one line?
Like `readonly working_dir="$(mktemp --directory)"`
Or is that harder to read? Seems like maybe it is?

Reply:
There's a sneaky problem with doing both in the same line - the exit code from `mktemp` is silently ignored! [Reference](https://github.com/koalaman/shellcheck/wiki/SC2155).

Reply:
Oh! Sneaky. :)